### ISSUE TITLE
Expose TabView system styles

### ISSUE THREAD

**Issue by hshristov** [NativeScript/NativeScript#260]
Provide an API for using system default TabBar items.
For iOS use system provided styles (e.g. UITabBarSystemItem and UIImageRenderingModeAlwaysOriginal).
For Android provide same icons in the default project template.

**Comment by jadsonlourenco** [comment]
Yes, we need this, because some apps that use TabView has icons too, so, if we can get the "native icons" and style the TabView will be great.

In this article (http://developer.telerik.com/featured/customizing-ios-navigation-bar-status-bar-nativescript/) we can see how stylize the NavigationBar, using JS, but I try do the same on Tabview and not work: 
```
var tabView1 = view.getViewById(page, "tabView1");
tabView1.barTintColor = UIColor.colorWithRedGreenBlueAlpha(0, 0.24, 0.45, 1);
```
Also if we can get the "native" properties, for example the "animated" property: ```animated: Bool``` (https://developer.apple.com/library/ios/documentation/UIKit/Reference/UITabBarController_Class/#//apple_ref/occ/instp/UITabBarController/customizableViewControllers).

Thanks!

**Comment by manijak** [comment]
Is this issue stil Open? It is possible to use native properties to style the TabView Bar but it is very limited. iOS limitation I guess? Is there anything we can do to smooth this out?

Having a hard time to style the not-selected tabView buttons (icon & text). My TabBar is blue, selected tintColor is White. But the non-selected ones are grey, and barely visible. No way to change them it seems.

**Comment by rjayako** [comment]
@jadsonlourenco were you able to get the tutorial link to work on iOS? Having a hard time changing the status bar colour on iOS. Can you post up a gist if you having a working demo?

**Comment by jadsonlourenco** [comment]
@rjayako sorry but I don't user NativeScript anymore - and don't solve this issue. I use React Native, the community is more active and help new devs, like me. Good luck!

**Comment by vakrilov** [comment]
@manijak @jadsonlourenco 
Being able to set the selected and non-selected colors of the tab-bar-items is something we are currently working on and will probably get into the next release. Actually, we already have it in the master branch [here](https://github.com/NativeScript/NativeScript/blob/6a9f212044ae5b62b522e03e6ac8574be8857d8d/ui/tab-view/tab-view.ios.ts#L308-L342).

**Comment by rclai** [comment]
What about the background color and top border color?

**Comment by rclai** [comment]
I was able to style everything except for the normal state image color, which I can't for the life of me figure out because I see that you're using the render original mode for the image, which is supposed to do the trick, but it is still gray :angry:

**Comment by rclai** [comment]
Here's the problem `renderingMode` is a read-only property: https://developer.apple.com/library/ios/documentation/UIKit/Reference/UIImage_Class/#//apple_ref/occ/instp/UIImage/renderingMode

**Comment by burkeholland** [comment]
Is there a way to do this with CSS instead of setting the `tabsBackgroundColor` directly on the element?

**Comment by manijak** [comment]
The way its now, no. But if the TabView included "TabViews" and "TabButtons" elements, then perhaps we could :-)

And btw, RenderMode Original means that custom colors don't apply. Images are used "as is", and their original color is shown. So you need two sets of images, one for each state.

**Comment by burkeholland** [comment]
I did notice per https://github.com/NativeScript/NativeScript/issues/767 that you can totally change the color of the text and items, but you have to do it as a property. It would be much easier to create re-usable themes if it was possible to change those items via class. 

I suppose we would have to provide a psuedo-class for the `selectedColor`. Perhaps `:active`?

**Comment by hamorphis** [comment]
@hshristov Include this in the 2.5 planning.

**Comment by dlucidone** [comment]
any progress on this?

**Comment by berardo** [comment]
Still open? :(

**Comment by alfrednerstu** [comment]
Would also love to see support for the native iOS icons as showed here: https://developer.apple.com/design/human-interface-guidelines/ios/icons-and-images/system-icons/#tab-bar-icons
